pkg/lib/listing.less

```diff
@@ -33,55 +33,55 @@ table.listing-ct > thead td > a {
     padding: @listing-ct-padding 0 @listing-ct-padding;
 }
 
-table.listing-ct thead th {
+table.listing-ct > thead th {
```

Comment:
Checked in Vagrant - other usage of the listing component seem to be untouched (i.e. on the "React Patterns" page)